test_dagmc_stats.py

```diff
@@ -99,3 +91,39 @@ def test_get_surfaces_per_volume():
     for eh in range(known_volumes.size()):
         surfs = my_core.get_child_meshsets(known_volumes[eh]).size()
         assert(surfs == s_p_v_data[eh])
+
+
+def test_get_triangle_aspect_ratio():
+    """
+    Tests part of the get_triangle_aspect_ratio function
+    """
+    test_input2 = "single-cube.h5m"
+    my_core = core.Core()
+    my_core.load_file(test_input2)
+    root_set = my_core.get_root_set()
+
+    exp = 1000*np.sqrt(2)/400/(10-5*np.sqrt(2))
+    obs = dagmc_stats.get_triangle_aspect_ratio(my_core, root_set)
+    assertAlmostEqual(exp,obs)
+    
+    exp = len(dagmc_stats.get_triangle_aspect_ratio(my_core, root_set))
```

Comment:
I think these two are backwards, at least in terms of the standard interpretation of `exp` and `obs`. 

Since you are testing `get_triangle_aspect_ratio`, its return value should be the observed result.

Reply:
I am sorry. I will also fix the backwards in test_get_area_triangle function together on Friday.